spec/core/data_structures.md

```diff
@@ -147,10 +147,10 @@ NOTE: that this is more specifically the consensus version and doesn't include i
 P2P Version. (TODO: we should write a comprehensive document about
 versioning that this can refer to)
 
-| Name  | type   | Description                                                                                                     | Validation                                                                                                         |
```

Comment:
Can we reword the previous paragraph?

Maybe we should only mention that the p2p version is not part of this `Version` field. The p2p version is only relevant at the p2p communication layer.